README.md

```diff
@@ -31,7 +24,13 @@ The GitHub integration for Slack gives you and your teams full visibility into y
 --------
 ## Installing the GitHub integration for Slack
 ### Requirements
-This app officially supports GitHub.com (which includes our GitHub Enterprise cloud-hosted offering) and Slack.com, but the team plans to support GitHub Enterprise Server (our self-hosted product) in the future.
+#### GHEC - GA
+This app officially supports GitHub.com (which includes our GitHub Enterprise cloud-hosted offering) and Slack.com. 
+
+#### GHES - Private preview
```

Comment:
Can add this subheading as a part of index nested under requirements.

Reply:
I think we are not doing 2nd level nested listing in the index. So it should be okay.